libandroid-navigation/src/test/java/com/mapbox/services/android/navigation/v5/navigation/FasterRouteDetectorTest.java

```diff
@@ -116,16 +116,8 @@ public void onNullRouteProgressPassed_shouldCheckFasterRouteIsFalse() throws Exc
 
```

Comment:
Normally is good to structure the tests following the triple A 👀 https://speakerdeck.com/guardiola31337/elegant-unit-testing-mobilization-2016?slide=40 http://wiki.c2.com/?ArrangeActAssert
If the `A`rrange part gets too long we should create factory or builder methods.

This applies to other modified tests included in this PR.